pwndbg/commands/next.py

```diff
@@ -36,6 +36,13 @@ def nextcall(*args):
 
 @pwndbg.commands.Command
 @pwndbg.commands.OnlyWhenRunning
+def nextsymbol(*args):
```

Comment:
The easiest way is to just not use regex at all, and let GDB do symbol lookups for you, by making this command a `ParsedCommand`.